common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -179,12 +179,17 @@ def create_modulestore_instance(
     else:
         disabled_xblock_types = ()
 
+    xblock_field_data_wrappers = [
+        load_function(path).wrap for path in settings.XBLOCK_FIELD_DATA_OVERRIDE_CLASSES
```

Comment:
(You could use the `"path.to.module:Class.function"` signature that the test finders use, for instance)